Mollier/SAM.Core.Mollier.UI/Forms/MollierForm.cs

```diff
@@ -572,19 +582,35 @@ private void a4LandscapeToolStripMenuItem_Click(object sender, EventArgs e)
         private void PointsCheckBox_CheckedChanged(object sender, EventArgs e)
         {
             MollierControlSettings mollierControlSettings = MollierControl_Main.MollierControlSettings;
+
             if (PointsCheckBox.Checked)
             {
-                mollierControlSettings.FindPoint = true;
-                mollierControlSettings.Percent = 0.4;
-                mollierControlSettings.FindPointType = ChartDataType.Enthalpy;
+                List<UIMollierPoint> mollierPoints = MollierControl_Main.UIMollierPoints;
+                if (mollierPoints == null || mollierPoints.Count < 4)
+                {
+                    MessageBox.Show("The minimum number of points on the chart required to run this method is 4.", "Error");
+                    PointsCheckBox.Checked = false;
+                    PercentPointsTextBox.Visible = false;
+                    PointsLabel.Visible = false;
+                    ColorPointComboBox.Visible = false;
+                }
+                else
+                {
+                    mollierControlSettings.FindPoint = true;
+                    mollierControlSettings.Percent = 0.4;
```

Comment:
If you call something Percent the it is usually value between 0-100. You you want value between 0-1 then it should be called Factor